core/src/test/java/com/backblaze/b2/client/B2StorageClientFactoryPathBasedImplTest.java

```diff
@@ -18,11 +19,10 @@
 
 
     @Test
-    public void testDefaultFactory_failsBecauseTestEnvironmentDoesntIncludeHttpClientJars() {
+    public void testDefaultFactory_failsSucceedsTestEnvironmentIncludesUrlBasedImplementation() {
```

Comment:
would you please change "failsSucceeds" to just "succeeds", please?  (i'm pretty sure that's my mistake!)